.circleci/config.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       - image: postgres:9.6
         environment:
           POSTGRES_USER: circleci
+          POSTGRES_PASSWORD: password
```

Comment:
This is not strictly related to the PR, but CircleCI was failing due to psql config stuff, so I just figured it would be easier to add in the same PR, happy to move it out if anybody is concerned

Reply:
Nice value add! :)